cli/cmd/cluster_port_rm.go

```diff
@@ -0,0 +1,32 @@
+package cmd
+
+import (
+	"github.com/replicatedhq/replicated/cli/print"
+	"github.com/spf13/cobra"
+)
+
+func (r *runners) InitClusterPortRm(parent *cobra.Command) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:  "rm",
+		RunE: r.clusterPortRemove,
+		Args: cobra.ExactArgs(1),
+	}
+	parent.AddCommand(cmd)
+
+	cmd.Flags().IntVar(&r.args.clusterPortRemovePort, "port", 0, "Port to remove")
+	cmd.Flags().StringArrayVar(&r.args.clusterPortRemoveProtocols, "protocol", []string{"http"}, "Protocol to remove")
+
```

Comment:
Probably a good idea. It's validated in the API, but debugging API failures is a pain.
Since we want to support additional protocols, I think ensuring there's at least 1 protocol is a good validation at this time.